### PR TITLE
Remove appraisals dependency (just use Gemfile)

### PR DESCRIPTION
**Pull request by lwe**
Appraisals basically just creates additional gemfiles in the gemfiles/ directory. These should be simple enough to manage without an additional gem.
- remove appraisals dev dependency
- reduce number gemfiles:
  - ✅ Gemfile (latest rails, i.e 5.x)
  - ✅ gemfiles/Gemfile.rails-4-2
- configure travis to exclude ruby <2.3 tests for Rails 5

/cc @oliverklee please review, thanks

### CODE REVIEW

**Comment by lwe** [comment]
JFYI, tests fail due to the things you are fixing in #40 - so I'll keep them failing if that's okay for you.

**Comment by oliverklee** [comment]
I'd like to first get the tests to run before we do any other changes, thus allowing us to see that we don't break anything with this change.

**Comment by lwe** [comment]
👌 wil rebase once we have fixed the tests

**Comment by oliverklee** [comment]
Great, thanks! (And I'm also happy to learn the non-Appraisal way of having multiple Rails versions for tests.)

**Comment by lwe** [comment]
👍 ready to be reviewed and merged.

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
-
 cache: bundler
-
 rvm:
   - 2.0.0
   - 2.1
   - 2.2.6
   - 2.3.3
-
+  - 2.4.0
```
Comment (oliverklee) [inline .travis.yml]: I'd prefer to have this in a separate PR (we already have #45 for this) so we can revert things separately if necessary. It also will make clearer what commit changed what.

File: .travis.yml [hunk 2]
```diff
@@ -1,26 +1,23 @@
 sudo: false
 
 language: ruby
-
 cache: bundler
-
```
Comment (oliverklee) [inline .travis.yml]: Any reason why you're removing these two blank lines?

**Comment by oliverklee** [comment]
I'd prefer to keep a 5_0 gemfile to make it explicit in the build matrix and the builds that we're testing Rails 5 there. What do you think?

File: .travis.yml (hunk 2 above)
Comment (lwe) [inline .travis.yml]: was not intentional, will fix

File: .travis.yml (hunk 1 above)
Comment (lwe) [inline .travis.yml]: k

**Comment by lwe** [comment]
Personally I disagree, because it's kind of duplicate information for what is in the `Gemfile` at the project root. How I see the structure is:

- gemfiles: old / "legacy" releases that are still supported
- (maybe: gemfiles: upcoming beta releases, e.g. rails 6 beta that is supported)
- `./Gemfile`: current stable Rails

**Comment by oliverklee** [comment]
Hm, what would you think of this?

- one gemfile for 4_2
- one gemfile for 5_0
- Gemfile for "latest" (e.g. Rails trunk)

Then we would have both consistent naming as well as tests with the latest Rails version, making us notice incompatibilities with upcoming versions early on.

**Comment by oliverklee** [comment]
When #52 is merged, the change of gemfiles also needs to be reflected in the RuboCop call.

**Comment by lwe** [comment]
@oliverklee, all right updated: check again.

**Comment by lwe** [comment]
Mhhpff, build seems to fail, because of rubocop + Ruby version, wtf, @oliverklee any idea why?

**Comment by oliverklee** [comment]
I think it has to do with the rainbow gem. I'll have a look at it.

File: Gemfile [hunk 3]
```diff
@@ -1,3 +1,3 @@
-source :rubygems
+source 'https://rubygems.org'
```
Comment (oliverklee) [inline Gemfile]: Why this change?

File: gemfiles/Gemfile.rails-4-2 [hunk 4]
```diff
@@ -0,0 +1,6 @@
+source 'http://rubygems.org'
```
Comment (oliverklee) [inline gemfiles/Gemfile.rails-4-2]: Can/should we also use source :rubygems here?

File: gemfiles/Gemfile.rails-5-0 [hunk 5]
```diff
@@ -0,0 +1,5 @@
+source 'http://rubygems.org'
```
Comment (oliverklee) [inline gemfiles/Gemfile.rails-5-0]: Can/should we also use source :rubygems here?
